diagnostic_aggregator/README.md

```diff
@@ -0,0 +1,186 @@
+# Overview
+This package contains the `aggregator_node`.
+It listens to the [`diagnostic_msgs/DiagnosticArray`](https://index.ros.org/p/diagnostic_msgs) messages on the `/diagnostics` topic and aggregates and published them on the `/diagnostics_agg` topic.
+
+## Example
+One example of how to use this package is to aggregate the diagnostics of a robot.
+Aggregation means that the diagnostics of the robot are grouped by various aspects, like their location on the robot, their type, etc.
+This will allow you to easily see which part of the robot is not working properly.
```

Comment:
I propose to move this explanation to the previous section and to start the example section directly with 'In our example ...'